continuous_integration/appveyor/environment.yaml

```diff
@@ -18,7 +18,7 @@ dependencies:
   - lz4
   - psutil
   - pytables
-  - fsspec>=0.5.1
+  - fsspec==0.6.0
```

Comment:
This appears to be `==` instead of `>=` as is the case in other requirements here. Is that intended?

Reply:
I was under the impression this was to test against both older and newer (future) supported releases of `fsspec` (similar to how we test different NumPy and pandas versions on different Travis builds). I.e. the Python 3.6 build makes sure `fsspec==0.6.0` still works while the Python 3.7 build will test newer releases as they are published. 

That said, we tended to update our minimum `fsspec` version frequently to the latest release, so maybe my understanding isn't correct here

Reply:
That would make sense.

Not sure either. Figured it was worth asking. Thanks for the answer. 🙂

Reply:
Yep, that was my intention. Thanks for checking.

Reply:
Thanks for confirming 🙂